test/gpu/nvidia_test.go

```diff
@@ -14,35 +15,41 @@ import (
 	"github.com/aws/amazon-cloudwatch-agent-test/test/test_runner"
 )
 
+var useE2EMetrics = flag.Bool("useE2EMetrics", false, "Use E2E metrics mapping which uses latest build CWA")
```

Comment:
Do we have test cases that aren't using the latest build?

Reply:
the integ test does not use the latest build

Reply:
Which one? Shouldn't all of these tests be using the latest build?

Reply:
It does look like it using the latest build but not sure why it isn't generating the right metrics(It might be missing a workload). We need to dive deeper